datahub-web-react/src/app/identity/user/SelectRole.tsx

```diff
@@ -25,7 +25,7 @@ const RoleIcon = styled.span`
 export default function SelectRole({ user, userRoleUrn, selectRoleOptions, refetch }: Props) {
     const [isViewingAssignRole, setIsViewingAssignRole] = useState(false);
     const [roleToAssign, setRoleToAssign] = useState<DataHubRole>();
-
+    const [roleToAssignEmpty] = useState<DataHubRole>();
```

Comment:
Can we remove this one from this particular PR? I don't see these changes discussed in PR description!